terraform-import-github-org.sh

```diff
@@ -183,47 +197,27 @@ get_team_ids () {
 }
 
 get_team_repos () {
-  for PAGE in $(limit_team_pagination); do
+  local team_data="$( curl -s "${API_URL_PREFIX}/teams/${TEAM_ID}?access_token=${GITHUB_TOKEN}" )"
+  local team_name="$( jq -r .name <<< "${team_data}" )"
+  local team_slug="$( jq -r .slug <<< "${team_data}" )"
+  local perms=
+  local team_repo_data=
 
+  for PAGE in $(limit_team_pagination); do
     for i in $(curl -s "${API_URL_PREFIX}/teams/$TEAM_ID/repos?access_token=$GITHUB_TOKEN&page=$PAGE&per_page=100" | jq -r 'sort_by(.name) | .[] | .name'); do
-    
-    TERRAFORM_TEAM_REPO_NAME=$(echo $i | tr  "."  "-")
-    TEAM_NAME=$(curl -s "${API_URL_PREFIX}/teams/$TEAM_ID?access_token=$GITHUB_TOKEN" | jq -r .name)
-    TEAM_SLUG=$(curl -s "${API_URL_PREFIX}/teams/$TEAM_ID?access_token=$GITHUB_TOKEN" | jq -r .slug)
-
-    ADMIN_PERMS=$(curl -s "${API_URL_PREFIX}/teams/$TEAM_ID/repos/$ORG/$i?access_token=$GITHUB_TOKEN" -H "Accept: application/vnd.github.v3.repository+json" | jq -r .permissions.admin )
-    PUSH_PERMS=$(curl -s "${API_URL_PREFIX}/teams/$TEAM_ID/repos/$ORG/$i?access_token=$GITHUB_TOKEN" -H "Accept: application/vnd.github.v3.repository+json" | jq -r .permissions.push )
-    PULL_PERMS=$(curl -s "${API_URL_PREFIX}/teams/$TEAM_ID/repos/$ORG/$i?access_token=$GITHUB_TOKEN" -H "Accept: application/vnd.github.v3.repository+json" | jq -r .permissions.pull )
-  
-    if [[ "$ADMIN_PERMS" == "true" ]]; then
-      cat >> github-teams-$TEAM_SLUG.tf << EOF
-resource "github_team_repository" "$TEAM_SLUG-$TERRAFORM_TEAM_REPO_NAME" {
-  team_id    = "$TEAM_ID"
-  repository = "$i"
-  permission = "admin"
-}
-
-EOF
-    elif [[ "$PUSH_PERMS" == "true" ]]; then
-      cat >> github-teams-$TEAM_SLUG.tf << EOF
-resource "github_team_repository" "$TEAM_SLUG-$TERRAFORM_TEAM_REPO_NAME" {
-  team_id    = "$TEAM_ID"
-  repository = "$i"
-  permission = "push"
-}
-
-EOF
-    elif [[ "$PULL_PERMS" == "true" ]]; then
-      cat >> github-teams-$TEAM_SLUG.tf << EOF
-resource "github_team_repository" "$TEAM_SLUG-$TERRAFORM_TEAM_REPO_NAME" {
-  team_id    = "$TEAM_ID"
-  repository = "$i"
-  permission = "pull"
-}
-
-EOF
-    fi
-    terraform import github_team_repository.$TEAM_SLUG-$TERRAFORM_TEAM_REPO_NAME $TEAM_ID:$i
+      team_repo_data="$( curl -s "${API_URL_PREFIX}/teams/${TEAM_ID}/repos/${ORG}/${i}?access_token=${GITHUB_TOKEN}" -H "Accept: application/vnd.github.v3.repository+json" )"
+      if [ "$( jq -r .permissions.admin <<< "${team_repo_data}" )" == true ] ; then
+          perms=admin
+      elif [ "$( jq -r .permissions.push <<< "${team_repo_data}" )" == true ] ; then
+          perms=push
+      elif [ "$( jq -r .permissions.pull <<< "${team_repo_data}" )" == true ] ; then
+          perms=pull
+      else
+          continue
```

Comment:
Again we continue because if not things don't exactly make sense (what permission would we set?)